app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.NoteApp">
+        <activity
+            android:name=".ShowNote"
+            android:parentActivityName=".MainActivity" />
```

Comment:
why to add MainActivity as parent?

Reply:
In order to enable "back" button in ShowNote action bar